app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,573 @@
+class StatsController < ApplicationController
+    def donation_totals
+        @result = ActiveRecord::Base.connection.execute(donation_query())
+        return @result
+      end
+
+    def gam_count
+        @GiftCardDetail = GiftCardDetail.where("single_use": true)
+        return @GiftCardDetail.length + 5140
```

Comment:
what's this 5140? Can we add a comment to explain?

Reply:
this is a bit of a magic number,  the gam spreadsheet and the website shows "13818" meals added. I believe when i did the query the database only has 8000 something meals. 

So the idea of 5140 was the assumed untracked meals that were not bought directly through the GAM page so 

DB_total + magic_constant = GAM total 
8632 + 5140 = 13818 

https://docs.google.com/spreadsheets/d/1gLPrM6E6JA-pkyXGWTJzhJnR-HXydIQ95voQAxuWAGI/edit#gid=0

like for example noodlehouse was recently added so this number increased from 13318 to 13818

Reply:
Gotchya, thanks for the explanation! I guess this is the kind of number we can eventually delegate to a Google Sheet!